source/draft_api.rst

```diff
@@ -648,7 +708,7 @@ Request body:
       "project": "ganeti_web_manager",
       "activities": ["docs"],
       "notes":"Worked on documentation toward settings configuration.",
-      "issue_uri": "https://github.com/osuosl/ganeti_webmgr/issues/56",
+      "issue_uri":"https://github.com/osu-cass/whats-fresh-api/issues/56",
```

Comment:
Small nitpick: some of the fields have a space separating the key from the value, while others don't. It may not need to be fixed now since both are valid JSON, but it should be fixed at some point for consistency.